tests/utils/cloudwatch/index.js

```diff
@@ -0,0 +1,23 @@
+const AWS = require('aws-sdk');
+const region = require('../misc').testRegion;
+
+function putCloudWatchEvents(sources) {
+  const cwe = new AWS.CloudWatchEvents({ region });
+
+  const entries = [];
+  sources.forEach(source => {
+    entries.push({
```

Comment:
This construct could probably be presented better with `entries = sources.map(...)`